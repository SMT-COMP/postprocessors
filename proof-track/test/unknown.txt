0.00/0.00	; run solver
14.65/6.58	; checking proof
14.73/6.79	unknown
14.73/6.81	; done
14.73/6.81	EOF
