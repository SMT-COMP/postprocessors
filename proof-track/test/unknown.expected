starexec-result=starexec-unknown
reason=unknown
